doc/build.rst

```diff
@@ -161,6 +161,24 @@ To build with MinGW, type:
 
   cp make/mingw64.mk config.mk; make -j4
 
+Opening a can of worms - building XGBoost library for python for Windows with MinGW-w64
```

Comment:
Can we remove the phrase "Opening a can of worms" from the title, to keep it short?